biz.aQute.bndlib.tests/test/test/ProjectTest.java

```diff
@@ -1198,4 +1206,41 @@ public void testCopyRepo() throws Exception {
 		assertTrue(project.check());
 		assertTrue(ws.check());
 	}
+
+
+	@Test
+	public void testWarnOnDuplicateProperties(@InjectTemporaryDirectory("generated/test-testWarnOnDuplicateProperties")
```

Comment:
```suggestion
	public void testWarnOnDuplicateProperties(@InjectTemporaryDirectory
```

Don't pass this arg. It will default to a reasonable name consistent with other uses.